### PR TITLE
SUBMARINE-619. Refactor Readme

### PR DESCRIPTION
**Pull request by kevin85421**
### What is this PR for?
Refactor the Readme.md

### What type of PR is it?
[Improvement]

### Todos
Add some pictures to explain the workflow in Submarine 
(ex: data exploration --> prototyping --> training --> monitoring --> serving)

### What is the Jira issue?
https://issues.apache.org/jira/projects/SUBMARINE/issues/SUBMARINE-619?filter=allopenissues

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,30 +42,27 @@ _Theodore Levitt_ once said:
...
 #### Model Training (Experiment)
-
-- Can run experiment (training jobs) on prem, on cloud. Via easy-to-use User-Interfaces
-- Easy for Data-Scientist (DS) to manage training code and dependencies (Docker, Python Dependencies, etc.) .
-- ML-focused APIs to run/track experiment from Python SDK (notebook), REST API, and CLI.
-- Provide APIs to run training jobs by using popular frameworks (Standalone/Distributed TensorFlow/PyTorch/Hovorod).
-- Pre-packaged Training Template for Data-Scientists to focus on domain-specific tasks (like using DeepFM to build a CTR prediction model).
-- Support GPU and other compute speed-up devices.
-- Support running on K8s/YARN or other resource management system.
+- Run/Track distributed training `experiment` on prem or cloud via easy-to-use UI/ API/ SDK.
```
Comment (jiwq) [inline README.md:45]:
```suggestion
- Run/Track distributed training `experiment` on prem or cloud via easy-to-use UI/API/SDK.
```

**Review by jiwq — APPROVED** [review]
+1

File: README.md (hunk 1 above)
Comment (kevin85421) [reply README.md]: @jiwq Thank you for this recommendation! I have fixed it.

**Review by xunliu — APPROVED** [review]
LGTM
Will merge if no more comments.

**Pull request closed without merging**